### PR TITLE
[stable/drone] Fixes compatibility with 1.16

### PR DESCRIPTION
**Pull request by null93**
Signed-off-by: Rafael Grigorian <me@raffi.io>

#### What this PR does / why we need it:

Makes stable/drone compatible with 1.16. A quick overview of deprecated APIs can be found on their blog post: https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/drone/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 name: drone
 home: https://drone.io/
 icon: https://drone.io/apple-touch-icon.png
-version: 2.1.0
+version: 2.1.1
```
Comment (christian-roggia) [inline stable/drone/Chart.yaml]: Maybe bumping the MINOR is more appropriate as this change is not a simple PATCH.

Comment (null93) [reply stable/drone/Chart.yaml]: Sounds appropriate

**Comment by metal3d** [comment]
That PR generate an error on clusters that are running in 1.13

We are not all able to make the migration, could you please change with:

```
{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
apiVersion: networking.k8s.io/v1beta1
{{- else -}}
apiVersion: extensions/v1beta1
{{- end }}
```

For example, for ingress and so on ?

**Pull request merged**